frontend/src/components/shared/buttons/new-project-button.tsx

```diff
@@ -10,24 +10,32 @@ interface NewProjectButtonProps {
 
 export function NewProjectButton({ disabled = false }: NewProjectButtonProps) {
   const { pathname } = useLocation();
-
   const { t } = useTranslation();
 
   const startNewProject = t(I18nKey.CONVERSATION$START_NEW);
 
   return (
-    <TooltipButton
-      tooltip={startNewProject}
-      ariaLabel={startNewProject}
-      navLinkTo="/"
-      testId="new-project-button"
-      disabled={disabled}
-    >
-      <PlusIcon
-        width={24}
-        height={24}
-        color={pathname === "/" ? "#ffffff" : "#B1B9D3"}
-      />
-    </TooltipButton>
+    <StyledTooltip content={startNewProject} placement="right">
+      <NavLink
+        to="/"
+        data-testid="new-project-button"
+        aria-label={startNewProject}
+        tabIndex={disabled ? -1 : 0}
+        onClick={(e) => {
+          if (disabled) {
+            e.preventDefault();
+          }
+        }}
+        className={`button-base inline-flex items-center justify-center ${
+          disabled ? "pointer-events-none opacity-50" : ""
+        }`}
```

Comment:
```suggestion
        className={cn("button-base inline-flex items-center justify-center", {
          "pointer-events-none opacity-50": disabled,
        })}
```

Use our `cn` util: https://github.com/OpenHands/OpenHands/blob/b5758b160415455976de704e0ead4567b0c74c2d/frontend/src/utils/utils.ts#L11